src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -27,10 +27,12 @@ public sealed partial class QuicConnection : System.IDisposable
         public System.Threading.Tasks.ValueTask CloseAsync(long errorCode, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public System.Threading.Tasks.ValueTask ConnectAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public void Dispose() { }
-        public long GetRemoteAvailableBidirectionalStreamCount() { throw null; }
-        public long GetRemoteAvailableUnidirectionalStreamCount() { throw null; }
-        public System.Net.Quic.QuicStream OpenBidirectionalStream() { throw null; }
-        public System.Net.Quic.QuicStream OpenUnidirectionalStream() { throw null; }
+        public int GetRemoteAvailableBidirectionalStreamCount() { throw null; }
```

Comment:
Why was this changed? Is it limited to int.Max by QUIC itself?

Reply:
Msquic returns `ushort`:
https://github.com/dotnet/runtime/blob/51b6188ed36fea7ad03e24a36224a600cdad1c72/src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs#L338-L346
AFAIK, we usually go with signed types for APIs ==> `int`.

I'm aware that QUIC sends `long`-range values in [MAX_STREAMS frame](https://quicwg.org/base-drafts/rfc9000.html#name-max_streams-frames), but that's cumulative number of streams which msquic hides from us. This is probably the original motivation for the `long` type.

Once again, I'm not against keeping it `long`. We discussed it briefly in one of our meetings and the consensus was for `int` so went ahead with the change.